src/Adapter/MSTest.CoreAdapter/Execution/TestClassInfo.cs

```diff
@@ -48,8 +49,11 @@ public class TestClassInfo
             this.ClassType = type;
             this.Constructor = constructor;
             this.TestContextProperty = testContextProperty;
+            this.BaseClassInitializeMethodsDict = new Dictionary<MethodInfo, bool>();
```

Comment:
nit: maybe rename this to be `ExecutedBaseClassInitializeMethods` and declare it as a HashSet<string> since we only have a DeclaringType check?

Reply:
agree, already updated.